cmd/webhook/main.go

```diff
@@ -130,14 +130,19 @@ func main() {
 		serviceName = "tekton-pipelines-webhook"
 	}
 
+	secretName := os.Getenv("WEBHOOK_SECRET_NAME")
+	if secretName == "" {
+		secretName = "webhook-certs" // #nosec
```

Comment:
Why is #nosec needed?

Reply:
Because golint warns about hard coded credentials if not set.

Reply:
@dibyom see https://securego.io/docs/rules/g101.html

Reply:
TIL! Thanks!